### DISCUSSION TITLE
Allow a task to fail (slack notification)

### DISCUSSION THREAD

**Discussion by midweste** [deployphp/deployer#3852] [Help needed]
We have a couple notifications that happen on deployment and occasionally the api endpoint takes longer than 5s to respond and deployer considers it a failed deployment.  Is there any way to allow a task to fail and not count that as a failed deployment step?

It's not a huge deal if the slack notification (or other) fails, we don't want it to run deploy;failed afterward.

Thanks!

**Comment by midweste** [comment]
Anyone have any ideas here?

**Reply by antonmedv** [reply]
Can you use try catch inside you tasks?

**Comment by antonmedv** [comment]
This should be possible:
```php
task("my-task", function () {
    try {
       invoke("other-task");
    } catch (e) {
    }
});
```

**Reply by midweste** [reply]
I'll test it out, thank you!
